Update docs for Ballerina Swan Lake Preview 1 (#61)
ballerina-platform/module-ballerina-java.jms: README.md

       }
   }
}

function createListener(jms:Connection connection) returns  jms:MessageConsumer {
    jms:Session session = checkpanic connection->createSession({acknowledgementMode: "AUTO_ACKNOWLEDGE"});
    jms:Destination queue = checkpanic session->createQueue("MyQueue");
    jms:MessageConsumer consumer = checkpanic session->createConsumer(queue);
    return consumer;
}
```

> ## Adding necessary dependencies 

Reviewer: ```suggestion
## Adding the required dependencies 
```

> Add the required dependencies to `Ballerina.toml` based on the broker that you're trying to connect to. 

Reviewer: ```suggestion
Add the required dependencies to the `Ballerina.toml` file based on the broker that you're trying to connect to. 
```

> To run the given examples using `Apache ActiveMQ` please add the following. 

Reviewer: ```suggestion
 Add the configurations below to run the given examples using `Apache ActiveMQ`. 
```
